libs/dh/shared/data-access-graphql/src/lib/graphql/get-grid-areas.graphql

```diff
@@ -6,5 +6,6 @@ query GetGridAreas {
     displayName
     validTo
     validFrom
+    includedInCalculation
```

Comment:
nitpick: a bit odd to include a page specific property to a "global" query but I don't have a better suggestion right now.